src/lib/CoinbasePro/Authenticated/Orders.hs

```diff
@@ -112,22 +124,26 @@ data Order = Order
     , executedValue :: Maybe ExecutedValue
     , status        :: Status
     , settled       :: Bool
+    , stop          :: Maybe StopLossSide
+    , stopPrice     :: Maybe Price
     } deriving (Eq, Show)
 
 
 deriveJSON defaultOptions {fieldLabelModifier = filterOrderFieldName . snakeCase} ''Order
 
 
 data PlaceOrderBody = PlaceOrderBody
-    { bClientOid :: Maybe ClientOrderId
-    , bProductId :: ProductId
-    , bSide      :: Side
-    , bSize      :: Size
-    , bPrice     :: Price
-    , bPostOnly  :: Bool
-    , bOrderType :: Maybe OrderType
-    , bStp       :: Maybe STP
-    , bTif       :: Maybe TimeInForce
+    { bClientOid   :: Maybe ClientOrderId
+    , bProductId   :: ProductId
+    , bSide        :: Side
+    , bSize        :: Size
+    , bPrice       :: Price
+    , bPostOnly    :: Bool
+    , bOrderType   :: Maybe OrderType
+    , bStp         :: Maybe STP
+    , bTimeInForce :: Maybe TimeInForce
```

Comment:
This field should be left as `bTif`. TIF is a well known acronym for time in force within the trading industry.

Reply:
I don't care what fields are called, I am just trying to minimize possible errors. I was following the coinbase API as directly as possible. "time_in_force" is what they have documented as the request parameter: https://docs.pro.coinbase.com/#place-a-new-order

Reply:
It seemed like the current field name was incorrect? Probably I am missing something.

Reply:
TIF is an appropriate abbreviation and I'd prefer if that was used. So 'bTif'.

Thanks!